app-starter/static/app-conf.json

```diff
@@ -64,7 +64,14 @@
         "name": "Name",
         "email": "Email",
         "website": "Website"
-      }
+      },
+      "selectStyle": {
+        "radius": 10,
+        "strokeColor": "gray",
+        "strokeWidth": 5,
+        "fillColor": "rgb(255, 255, 0, 0.2)"
+      },
+      "doAppendSelectStyle": true
```

Comment:
Maybe better `appendSelectStyle`

Reply:
@chrismayer I intentionally called it `doAppendSelectStyle` to make clear that the value should be a Boolean and not a style definition object. But we can also rename it.

Reply:
Just let me know your preference than I can change it.

Reply:
Right, so it is clearer. Keep it as it is.